tools/mo/openvino/tools/mo/front/tf/loader.py

```diff
@@ -334,6 +334,10 @@ def convert_to_pb(argv: argparse.Namespace):
             isinstance(argv.input_model, str):
         return None
 
+    # Saved Model format and MetaGraph format is supported without freezing
+    if argv.saved_model_dir or argv.input_meta_graph:
```

Comment:
Is this really needed here? Can you just not call `convert_to_pb` for meta and sm?

Reply:
I don't know how it works here, copy/pasted from original solution :) If you know better way - could you point on it?

Reply:
Checkpoint is remained. Once checkpoint is implemented, we will remove `convert_to_pb` at all.

Reply:
So I am leaning to hide this inside this function